tools/spectral/ipa/__tests__/eachResourceHasGetMethod.test.js

```diff
@@ -0,0 +1,86 @@
+import testRule from './__helpers__/testRule';
+import { DiagnosticSeverity } from '@stoplight/types';
+
+testRule('xgen-IPA-104-resource-has-GET', [
```

Comment:
can you add a case with nested resource?
`/groups/{groupId}/clusters`? and a singleton like `/groups/{groupId}/settings`?

Reply:
Done!